src/index.js

```diff
@@ -0,0 +1 @@
+( ()=> {  console.log("hey Riders");})();
```

Comment:
Great work but instead of this console, you can consider initializing react and render something.

Reply:
@Niyitangasam, I worked on this issue.